stetho/src/main/java/com/facebook/stetho/inspector/elements/ActivityNodeDescriptor.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.elements;
+
+import android.app.Activity;
+import android.view.Window;
+
+final class ActivityNodeDescriptor extends ChainedNodeDescriptor<Activity> {
+  public ActivityNodeDescriptor() {
+  }
+
+  // TODO: do we have to worry about listening for when the Activity's Window transitions
```

Comment:
It goes from null to non-null ... when an Activity is first created, it isn't yet plugged into a Window, right? And yet there's nothing here hooking in to listen to when the Window is attached. I think we'll be able to rely on the tree diffing stuff though for that.